source/docs/advanced/payments/front-commerce-payments.md

```diff
@@ -0,0 +1,71 @@
+
+Payment platforms integrated as a Front-Commerce Payment are compatible with every eCommerce platform that Front-Commerce supports. They also are aimed at being as frictionless as possible for Customers, so payment inputs are usually embedded in the checkout page to prevent redirecting Customers to third-party providers. Per their nature follow the [Direct Order payment workflow](/docs/advanced/payments/payment-workflows.html#Direct-Order).
+
+## Supported Payment platforms
+
+Front-Commerce Payments are currently available for the platforms below, learn how to install each one of them from the related documentation page:
+- [Stripe](/docs/advanced/payments/stripe.html#Front-Commerce-Payment)
+- [Paypal](/docs/advanced/payments/paypal.html#Front-Commerce-Payment)
+- [PayZen](/docs/advanced/payments/payzen.html#Front-Commerce-Payment)
+- [Ingenico](/docs/advanced/payments/ingenico.html#Front-Commerce-Payment)
+
+<blockquote class="info">
+  If you want to use a Payment platform not yet listed above, please [`contact us`](mailto:contact@front-commerce.com) so we can provide information about a potential upcoming native support for it.
+</blockquote>
+
+## How is a Front-Commerce Payment method integrated with eCommerce platforms
+
+Front-Commerce handle payments directly, without the need for a payment integration in your eCommerce system. However, **Front-Commerce provides a generic Payment method** tailored for your eCommerce platform so that you can still benefit from its order management features and workflows.
+
+From your eCommerce system's perspective, a payment made with Front-Commerce is integrated in a similar way than checks / wire payments (sometimes named "offline payment"). The difference is that it Front-Commerce will update orders statuses upon receiving automatic notifications from the payment service.
+
+## Implement a new Front-Commerce Payment method
+
+<blockquote class="wip">
+**Work In Progress** This section is not as detailed as we would love it to be. Please let us know if you need further information before we improve it.
+</blockquote>
+
+Front-Commerce allows you to implement your own payment method. New embedded payment methods have to be registered from a GraphQL module. Let’s create a new "PWAy" payment module for a fictive payment provider.
+
+1. [create a new GraphQL module](/docs/essentials/extend-the-graphql-schema.html#Create-a-new-GraphQL-module)
+2. add a depency upon `"Magento2/Checkout"`
+```js
+export default {
+  namespace: "Payments/PWAy",
+  dependencies: ["Magento2/Checkout"]
+};
+```
+3. register the payment method from the module’s [`contextEnhancer`](/docs/reference/graphql-module-definition.html#contextEnhancer-optional)
+```js
+export default {
+  namespace: "Payments/PWAy",
+  dependencies: ["Magento2/Checkout"],
+  contextEnhancer: ({ loaders }) => {
+    // [...] initialization here
+
+    const chargeTransactionForOrder = (paymentData, orderId) => {
```

Comment:
For the final doc, we won't put this method in the contextEnhancer and will instead use a loaders.js file.

Reply:
That's what the comment says inside the function. I'm not sure it would be worth it, since it will add an indirection that could make understanding more difficult.

Let's test it in interviews when we'll do some UX docs!

Reply:
In my experience, it's easier during trainings to always keep the same file structure. Even if it's not always super clear how it *works*, it at least helps to know where to look and they can compare with existing code.